frontend/src/components/jobSeekerSignup/StepTwoJobSeekerSignUp.js

```diff
@@ -136,12 +137,12 @@ function StepTwoJobSeekerSignUp({setValue, address, apt, city, state, zip, posit
     },[school, schoolEndDateMonth, schoolEndDateYear, degree, field])
     return ( 
         <>
-            <div className="signUpBG" style={{height: "1500px"}} >
+            <div className="signUpBG" >
                 <h1 className='text-center'>
-                    One More Step..
+                    Personal Information
                 </h1>
-            
-                <div className='whiteBG mx-auto mt-4' style={{width: '800px', height: '1250px'}}>
+                <div className="text-center">This information will be used to get you closer to the dream job!</div>
```

Comment:
Recommendation: Instead of ". . . to get you closer to the dream job", what about ". . . to get you closer to your dream job"

Reply:
Done, changed it to the reccomendation.